kfk_consumer.js

```diff
@@ -0,0 +1,55 @@
+//Avro
+var avro = require('avsc');
+var type = avro.parse(typeDescription);
+
+var avroSchemaIn = {
+    name: 'drone1_videostream',//or whatever we call this
+    type: 'record',
+    fields: [
+      {
+        name: 'id',
+        type: 'string'
+      }, {
+        name: 'timestamp',
+        type: 'double'
+      }, {
+        name: 'bytes',//img hopefully?
+        type: 'string'
+      }]
+  };
+
+  
+//Kafka Consumer (Unconfigured)
+var kafka = require('kafka-node');
+var HighLevelConsumer = kafka.HighLevelConsumer;
+var Client = kafka.Client;
+
+var client = new Client('localhost:2181');
+var topics = [{
```

Comment:
We will want it to poll for all topics and separate out the video stream topics, then setup consumer listeners for each.